src/index.js

```diff
@@ -1,10 +1,3 @@
-import "core-js/fn/array/from";
-import "core-js/fn/array/includes";
-import "core-js/fn/map";
-import "core-js/fn/object/get-own-property-symbols";
-import "core-js/fn/object/is";
-import "core-js/fn/object/keys";
-import "core-js/fn/set";
 import Helmet from "./Helmet";
```

Comment:
@doctyper @dhoward you guys think we should remove the index.js file as well, and change package.json to point directly to Helmet.js?  Would match Wildcat convention as well :-) just sayin'